locales/ES_es.json

```
@@ -0,0 +1,139 @@
+			},
+			{
+				"question": "¿Mi NNID actual funciona en Pretendo?",
+				"answer": "Tristemente, no. Los NNID no funcionan en Pretendo, ya que solo Nintendo almacena tus datos de usuario. Aunque una migración NNID-a-PNID es teóricamente posible, sería arriesgado y requiere datos de usuario sensibles que preferimos no almacenar."
+			},
+			{
+				"question": "¿Cómo uso Pretendo?",
+				"answer": "Pretendo no está actualmente en un estado adecuado para el uso público. Sin embargo, en cuanto esté listo podrás usar Pretendo simplemente cargando nuestro parcheador homebrew en tu consola."
+			},
+			{
+				"question": "¿Sabéis cuándo <determinada función/servicio> estará listo/a?",
+				"answer": "No. Muchos/as funciones/servicios de Pretendo son desarrolladas independientemente (por ejemplo, un desarrollador puede estar trabajando en Miiverse mientras otro se encarga de las Cuentas y los Amigos) y, por lo tanto, no podemos dar una ETA exacta de cuánto puede tardar en estar listo."
+			},
+			{
+				"question": "¿Pretendo funciona en Cemu/emuladores?",
+				"answer": "Pretendo está diseñado con el hardware de Wii U y 3DS en mente. Hasta ahora, el único emulador para esas consolas con soporte de NN es Cemu. Cemu no soporta oficialmente servidores personalizados, pero debería ser posible en un futuro llegar a usar Pretendo con Cemu.<br>A día de hoy, Pretendo no soporta Cemu."
+			},
+			{
+				"question": "Si estoy baneado en Nintendo Network, ¿seguiré baneado al usar Pretendo?",
+			"answer": "No tenemos acceso a los baneos de Nintendo Network motivo por el nadie estará baneado en nuestros servidores al principio. Sin embargo, tendremos reglas a seguir al acceder a nuestros servicios y no hacerlo resultará en un baneo."				
+			},
+			{
+				"question": "¿Pretendo soportará Wii/Switch?",
+				"answer": "La Wii ya tiene servidores en línea personalizados de parte de <a href=\"https://wiimmfi.de/\" target=\"_blank\">Wiimmfi</a>. No tenemos planeado trabajar con Switch ya que sus servidores son de pago y completamente diferentes a los de Nintendo Network."
+			},
+			{
+				"question": "¿Necesitaré modificar mi consola para conectarme?",
+				"answer": "Sí, necesitas modificar tu consola para conectarte. Sin embargo, en Wii U solo necesitas acceso al Homebrew Launcher (p.e. Haxchi, Coldboot Haxchi, o el exploit web). La información sobre como conectarse con una 3DS llegará posteriormente."
+			}
+		]
+	},
+	"credits": {
+		"title": "Equipo",
+		"text": "Conoce al equipo tras el proyecto",
+		"people": [
+			{
+				"name": "Jonathan Barrow (RedDucks)",
+				"caption": "Líder del proyecto y desarrollador principal",
+				"picture": "https://github.com/redduckss.png",
+				"github": "https://github.com/RedDuckss"
+			},
```

jon has a new username on github, please change to this:
```suggestion
			{
				"name": "Jonathan Barrow (jonbarrow)",
				"caption": "Líder del proyecto y desarrollador principal",
				"picture": "https://github.com/jonbarrow.png",
				"github": "https://github.com/jonbarrow"
			},
```